src/jail_rules.c

```diff
@@ -566,6 +573,13 @@ jail_rules_parse_section(
             jail_rules_add_path(data, local, TRUE, FALSE);
             g_free(local);
         }
+
+        /* If APPNAME.profile exists, it is implicitly pulled in */
+        {
```

Comment:
Need to make sure that it exists before calling `jail_rules_add_profile()` to avoid misleading `"XXX doesn't exist"` warnings for something that was never mentioned in the file being parsed.

Reply:
mm, I guess so. Earlier I had this use jail_rules_add_profile(..., required=false) but the semantics/results from that were not quite as what I expected i.e. this needs to be handled as "required, but only if it exists" kind of way ...

Reply:
added existence check and tried to make the code more like what there already was